plane_mcp/journey/tools/read.py

```
@@ -0,0 +1,246 @@
+        if priority:
+            query_params["priority"] = ",".join([p.lower() for p in priority])
+            
+        if states:
+            state_ids = [self.resolver.resolve_state(project_slug, s) for s in states]
+            query_params["state"] = ",".join(state_ids)
+            
+        if labels:
+            unresolved_labels = []
+            try:
+                existing_labels = client.labels.list(workspace_slug=workspace_slug, project_id=project_id).results
+                name_to_id = {l.name.lower(): l.id for l in existing_labels if l.name}
```

Ruff (0.15.9)

[error] Ambiguous variable name: `l`

(E741)